src/rpdk/core/contract/resource_client.py

```diff
@@ -362,17 +362,26 @@ def compare(self, inputs, outputs):
                         self.compare(inputs[key], outputs[key])
                     elif isinstance(inputs[key], list):
                         assert len(inputs[key]) == len(outputs[key])
-                        self.compare_list(inputs[key], outputs[key])
+                        is_ordered = self._schema["properties"][key].get(
+                            "insertionOrder", True
```

Comment:
Just FYI, we default to False for features like Drift Detection. I am fine with this for the purpose of contract tests though.